cloudstack_plugin/network.py

```diff
@@ -104,8 +104,15 @@ def create(ctx, **kwargs):
                     acl_type = acl.get('type')
 
                     for port in acl_ports:
+                        # Specifying port ranges "start_port:end_port"
+                        ports = str(port).split(':')
+                        start_port = int(ports[0])
+                        if len(ports) == 1:
+                            end_port = start_port
+                        elif len(ports) == 2:
+                            end_port = int(ports[1])
```

Comment:
maybe add 
```
else:
    raise NonRecoverableError(
        'port may be a single port or a port range in the format start_port:end_port')
```
In the scenario that a user puts a typo like "80:443:8080", instead of "80, 443, 8080" or even "80:443, 8080", this will raise an error instead of doing something unexpected.